src/app/[locale]/(default)/about/LabOps/page.tsx

```diff
@@ -0,0 +1,124 @@
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/ui/Accordion';
+import { Card } from '@/components/ui/Card';
+import { useTranslations } from 'next-intl';
+import { unstable_setRequestLocale } from 'next-intl/server';
+import Image from 'next/image';
+
+import * as React from 'react';
+import ClientLabOpsPage from './ClientPage'; // Ensure this path is correct
+
+export default function labopsPage({
+    params: { locale },
+  }: {
+    params: { locale: string};
+  }) {
+    unstable_setRequestLocale(locale);
+    const t = useTranslations('labops');
+
+    return (
+        <div>
+            <div className=' items-center flex flex-col'>
+                <div className='flex items-center justify-center w-full max-w-full max-h-[600px] sm:max-w-[400px] sm:max-h-[400px]'>
+                    <ClientLabOpsPage locale={locale} />
+                </div>
+            </div>
+            <h1 className='w-full mt-4 mb-5 dark:text-primary'> LabOps </h1>
+          <div className='flex w-full mt-5 mb-5'>
+            <p>
+                {t('aboutLabOps')}
+            </p>
+          </div>
+          <div className='mt-10 mb-10'>
+            <h3> FAQ'S </h3>
+            <Accordion type="single" collapsible className='w-full dark:text-foreground'>
+                <AccordionItem value='item-1'>
+                    <AccordionTrigger className='m-2'>
+                        hei
+                    </AccordionTrigger>
+                    <AccordionContent className='text-base m-2'>
+                        heheeh
+                    </AccordionContent>
+                </AccordionItem>
+                <AccordionItem value='item-2'>
+                    <AccordionTrigger className='m-2'>
+                        hei
+                    </AccordionTrigger>
+                    <AccordionContent className='text-base m-2'>
+                        heheeh
+                    </AccordionContent>
+                </AccordionItem>
+                <AccordionItem value='item-3'>
+                    <AccordionTrigger className='m-2'>
+                        hei
+                    </AccordionTrigger>
+                    <AccordionContent className='text-base m-2'>
+                        heheeh
+                    </AccordionContent>
+                </AccordionItem>
+            </Accordion>
+          </div>
+          <div className='w-full max-x-xs px-10'>
+            <div className='grid gap-6 sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-2 dark:text-primary'>
```

Comment:
You shouldnt need to do dark mode specific colors normally since it is handled automatically by css variables